tests/src/Functional/UiFieldTest.php

```diff
@@ -226,15 +226,19 @@ public function testFieldSaved() {
     // Asserts the debug mode of twig is enabled.
     $this->assertTrue(strpos($output->getContent(), '<!-- THEME HOOK: \'node\' -->') !== FALSE);
 
-    // Asserts that all Template Whisperer based suggestions are present.
+    // Asserts that all Page Template Whisperer based suggestions are present.
+    $this->assertTrue(strpos($output->getContent(), '* page--node--1--googlemap.html.twig') !== FALSE);
+    $this->assertTrue(strpos($output->getContent(), '* page--node--googlemap.html.twig') !== FALSE);
+
+    // Asserts that all Entity Template Whisperer based suggestions are present.
     $this->assertTrue(strpos($output->getContent(), '* node--article--googlemap.html.twig') !== FALSE);
     $this->assertTrue(strpos($output->getContent(), '* node--1--article--googlemap.html.twig') !== FALSE);
     $this->assertTrue(strpos($output->getContent(), '* node--article--full--googlemap.html.twig') !== FALSE);
     $this->assertTrue(strpos($output->getContent(), '* node--1--article--full--googlemap.html.twig') !== FALSE);
   }
 
   /**
-   * Tests the Node whitout Template saved, don't suggestion it.
+   * Tests the Node whiteout Template saved, don't suggestion it.
```

Comment:
```suggestion
   * Tests the Node without Template saved, don't suggestion it.
```

Reply:
argh, it looks like you already applied this change in the target branch.